Doc/whatsnew/3.9.rst

```diff
@@ -213,6 +213,12 @@ Removed
   instead.  The ``xml.etree.cElementTree`` module has been removed.
   (Contributed by Serhiy Storchaka in :issue:`36543`.)
 
+* The old :mod:`plistlib` API has been removed, it was deprecated since Python
+  3.4. Use the :func:`load`, :func:`loads`, :func:`dump`, and :func:`dumps`
+  functions. Additionally, the `use_builtin_types` parameter was removed,
```

Comment:
There is a CI failure due to the way `use_buitin_types` is written.  Please use double backticks.